orc8r/cloud/go/tools/accessc/handlers/add_existing.go

```diff
@@ -90,7 +93,7 @@ func addExisting(cmd *commands.Command, args []string) int {
 				log.Fatalf(
 					"Cannot parse certificate from '%s': %s\n", certFName, err)
 			}
-			err = certifier.AddCertificate(operator, certDERBlock.Bytes)
+			err = certifier.AddCertificate(context.Background(), operator, certDERBlock.Bytes)
```

Comment:
For all the accessc handlers: I'd imagined that each cmd should use a single ctx for the entirety of the cmd